src/test/java/com/exadel/discount/DiscountApplicationTests.java

```diff
@@ -0,0 +1,4 @@
+package com.exadel.discount;
+
+public class DiscountApplicationTests {
```

Comment:
empty class?

Reply:
Done, I forgot the annotation, thx

Reply:
I don't understand why do you need it even with annotation...